docs/Training-PPO.md

```diff
@@ -166,6 +166,19 @@ variables, this should be larger.
 
 Typical Range: `32` - `512`
 
+### (Optional) Visual Encoder Type
+
+`vis_encode_type` corresponds to the encoder type for encoding visual observations.
+Valid options include:
+* `default` (default): a simple encoder which consists of two convolutional layers
```

Comment:
Could we possibly call this one "simple" rather than default? I think for many problems it is actually too basic to be considered a good default.

Reply:
Good call, this sounds better.